shared/store/action-logger.js

```diff
@@ -15,13 +22,47 @@ const objToJS = state => {
   return newState
 }
 
-export const actionLogger = store => next => action => {
+const loggingStatSink: StatSink = {
+  label: 'Action Logger',
+  totalTime: 0,
+  totalActions: 0,
+  timings: [], // to calculate variance
+  startTime: 0,
+}
+
+const actionStatSink: StatSink = {
+  label: 'Actions',
+  totalTime: 0,
+  totalActions: 0,
+  timings: [], // to calculate variance
+  startTime: 0,
+}
+
+export const actionLogger = (store: any) => (next: any) => (action: any) => { // eslint-disable-line arrow-parens
   console.groupCollapsed && console.groupCollapsed(`Dispatching action: ${action.type}`)
 
   console.log(`Dispatching action: ${action.type}: ${JSON.stringify(action)} `)
```

Comment:
Just turn action logger off. We have a flag already

On Thursday, July 14, 2016, Gabriel Handford <notifications@github.com>
wrote:

> In shared/store/action-logger.js
> <https://github.com/keybase/client/pull/3463#discussion_r70908979>:
>
> > +  label: 'Action Logger',
> > +  totalTime: 0,
> > +  totalActions: 0,
> > +  timings: [], // to calculate variance
> > +  startTime: 0,
> > +}
> > +
> > +const actionStatSink: StatSink = {
> > +  label: 'Actions',
> > +  totalTime: 0,
> > +  totalActions: 0,
> > +  timings: [], // to calculate variance
> > +  startTime: 0,
> > +}
> > +
> > +export const actionLogger = (store: any) => (next: any) => (action: any) => { // eslint-disable-line arrow-parens
> >    console.groupCollapsed && console.groupCollapsed(`Dispatching action: ${action.type}`)
> >
> >    console.log(`Dispatching action: ${action.type}: ${JSON.stringify(action)} `)
>
> Stringify kind of makes it less cool in dev tools.. is there a way to not
> stringify when in DEV mode?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3463/files/e06d4da5af7db2cdfcc36eb23416bb042f557b1b#r70908979>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhUywPWaIrFY04D7fJ_ScXgrQfqnODks5qVt4pgaJpZM4JMA3T>
> .
>